src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -21,15 +19,11 @@
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a question to the address book. "
             + "Parameters: "
             + PREFIX_NAME + "NAME "
-            + PREFIX_PHONE + "PHONE "
-            + PREFIX_EMAIL + "EMAIL "
-            + PREFIX_ADDRESS + "ADDRESS "
+            + PREFIX_IMPORTANCE + "IMPORTANCE "
             + "[" + PREFIX_TAG + "TAG]...\n"
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_NAME + "John Doe "
-            + PREFIX_PHONE + "98765432 "
-            + PREFIX_EMAIL + "johnd@example.com "
-            + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
+            + PREFIX_IMPORTANCE + "1 "
```

Comment:
Future Note: this can be replaced using an enum but for now it is ok as it is not yet used.